WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/JetpackScanCoordinator.swift

```diff
@@ -132,4 +172,26 @@ class JetpackScanCoordinator {
     private struct Constants {
         static let refreshTimerInterval: TimeInterval = 5
     }
+
+
+    private struct Strings {
+        static let fixAllAlertTitleFormat = NSLocalizedString("Please confirm you want to fix all %1$d active threats", comment: "Confirmation title presented before fixing all the threats, displays the number of threats to be fixed")
+        static let fixAllSingleAlertTitle = NSLocalizedString("Please confirm you want to fix all %1$d active threat", comment: "Confirmation title presented before fixing a single threat")
+        static let fixAllAlertTitleMessage = NSLocalizedString("Jetpack will be fixing all the detected active threats.", comment: "Confirmation message presented before fixing all the threats, displays the number of threats to be fixed")
+        static let fixAllSingleAlertMessage = NSLocalizedString("Jetpack will be fixing all the detected active threat.", comment: "Confirmation message presented before fixing a single threat")
```

Comment:
Single threat string 👀 
```suggestion
        static let fixSingleAlertMessage = NSLocalizedString("Jetpack will be fixing the detected active threat.", comment: "Confirmation message presented before fixing a single threat")
```

Reply:
I'd been staring at strings for too long 😆 Fixed in: f67b8205b767420aec81abc1f6f6baff86188e82